extensions/wikia/CommunityPage/models/CommunityPageSpecialUsersModel.class.php

```diff
@@ -455,15 +458,19 @@ function () {
 
 				$botIds = $this->getBotIds();
 
-				$numberOfMembers = ( new WikiaSQL() )
+				$sqlData = ( new WikiaSQL() )
 					->SELECT()
 					->COUNT( 'DISTINCT rev_user' )->AS_( 'members_count' )
 					->FROM( 'revision' )
-					->AND_( 'rev_user' )->NOT_EQUAL_TO( 0 )
-					->AND_( 'rev_user' )->NOT_IN( $botIds )
-					->runLoop( $db, function ( &$numberOfMembers, $row ) {
-						$numberOfMembers = (int)$row->members_count;
-					} );
+					->AND_( 'rev_user' )->NOT_EQUAL_TO( 0 );
+
+				if ( ! empty($botIds) ) {
+					$sqlData->AND_( 'rev_user' )->NOT_IN( $botIds );
+				}
+
+				$numberOfMembers = $sqlData->runLoop( $db, function ( &$numberOfMembers, $row ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$numberOfMembers". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)